docs/presentations/articles/langchain.md

```diff
@@ -0,0 +1,99 @@
+# Langchain: Simplifying LLM-Powered Application Development
+Author: Tobias Brenner
+
+## TL;DR¶
```

Comment:
The `¶` character seems weird. Can you remove it throughout the document?

![image](https://github.com/pkeilbach/htwg-practical-nlp/assets/28002199/22ca4c6e-9883-4856-901c-0a568b8afbe2)

Reply:
@pkeilbach  The ¶ signs are removed now 😄